frontend/src/App.js

```diff
@@ -1,15 +1,25 @@
 import "./App.css";
-import ScrollToTop from "./utils/ScrollToTop";
+//import ScrollToTop from "./utils/ScrollToTop";
 import { BrowserRouter, Routes, Route } from "react-router-dom";
+//import LandingPage from "./pages/landingPage"
+import Register from "./pages/auth/register";
+import Login from './pages/auth/login';
+import Forgot_Password from './pages/auth/forgot_password';
 
 function App() {
   return (
-    <BrowserRouter>
-      <ScrollToTop />
+    <>
+       <BrowserRouter>
+        {/*<ScrollToTop />*/}
       <Routes>
-        <Route path="/" element={<>you're here</>} />
+      {/*<Route path="/" element={<LandingPage/>} />*/}
+      <Route path="/" element={<Register/>} />
```

Comment:
The Register component should not be rendered on the homepage. Let's reserve the `"/"` route for the HomePage component.